async-nats/tests/jetstream_tests.rs

```diff
@@ -232,6 +232,27 @@ mod jetstream {
             PublishErrorKind::WrongLastSequence
         );
 
+        context
+            .send_publish(
+                "baz".to_string(),
+                Publish::build().expected_last_subject_sequence(0),
+            )
+            .await
+            .unwrap()
+            .await
+            .unwrap();
+
+        // Should not error
+        context
+            .send_publish(
+                "baz".to_string(),
+                Publish::build().expected_last_subject_sequence(0),
```

Comment:
You forgot to change it back (during testing this was one if I observed c correctly)
```suggestion
                Publish::build().expected_last_subject_sequence(1),
```

Reply:
You're right, I missed that one. Fixed now!